docs/api/config-manager.md

```diff
@@ -0,0 +1,249 @@
+
+```ts
+loadSensorTypes(): Promise<void>;
+```
+
+Loads the sensor config which is used to lookup sensor types and scales.
+
+### `lookupSensorType`
+
+```ts
+lookupSensorType(sensorType: number): SensorType | undefined;
+```
+
+Looks up the configuration for a given sensor type
+
+> [!NOTE] `loadSensorTypes` must be used first.
+
+### `getSensorTypeName`
+
+```ts
+getSensorTypeName(sensorType: number): string;
+```
+
+Returns the label for a given sensor type.
+
+> [!NOTE] `loadSensorTypes` must be used first.
+
+### `lookupSensorScale`
+
+```ts
+lookupSensorScale(sensorType: number, scale: number): Scale;
+```
+
+Looks up a scale definition for a given sensor type
+
+> [!NOTE] `loadSensorTypes` and `loadNamedScales` must be used first.
+
+### `loadMeters`
+
+```ts
+loadMeters(): Promise<void>;
+```
+
+Loads the meters config which is used to lookup meter types and scales.
+
+### `lookupMeter`
+
+```ts
+lookupMeter(meterType: number): Meter | undefined;
+```
+
+Looks up the meter definition for a given meter type.
+
+> [!NOTE] `loadMeters` must be used first.
+
+### `getMeterName`
+
+```ts
+getMeterName(meterType: number): string;
+```
+
+Returns the meter label for a given meter type.
+
+> [!NOTE] `loadMeters` must be used first.
+
+### `lookupMeterScale`
+
+```ts
+lookupMeterScale(type: number, scale: number): MeterScale;
+```
+
+Looks up a scale definition for a given meter type
+
+> [!NOTE] `loadMeters` must be used first.
+
+### `loadDeviceClasses`
+
+```ts
+loadDeviceClasses(): Promise<void>;
+```
+
+Loads the device classes config which is used to lookup basic, generic and specific device classes.
+
+### `lookupBasicDeviceClass`
+
+```ts
+lookupBasicDeviceClass(basic: number): BasicDeviceClass;
+```
+
+### `lookupGenericDeviceClass`
+
+```ts
+lookupGenericDeviceClass(generic: number): GenericDeviceClass;
+```
+
+### `lookupSpecificDeviceClass`
+
+```ts
+lookupSpecificDeviceClass(generic: number, specific: number): SpecificDeviceClass;
+```
+
+### `loadDeviceIndex`
+
+```ts
+loadDeviceIndex(): Promise<void>;
+```
+
+Loads the device index which is internally used to lookup device configuration files.
+
+### `getIndex`
+
+```ts
+getIndex(): DeviceConfigIndexEntry[] | undefined;
+```
+
+Returns the previously loaded device index.
+
+> [!NOTE] `loadDeviceIndex` must be used first.
+
+### `lookupDevice`
+
+```ts
+lookupDevice(manufacturerId: number, productType: number, productId: number, firmwareVersion?: string): Promise<DeviceConfig | undefined>;
+```
+
+Looks up the definition of a given device in the configuration DB. It is not necessary to use `loadDeviceIndex` first.
+
+-   `manufacturerId`: The manufacturer id of the device
+-   `productType`: The product type of the device
+-   `productId`: The product id of the device
+-   `firmwareVersion`: If known, configuration for a specific firmware version can be loaded. If this is `undefined` or not given, the first matching file with a defined firmware range will be returned.
```

Comment:
I agree with @marcus-j-davies here you could point users to https://zwave-js.github.io/node-zwave-js/#/development/config-files?id=properties